includes/Post/HandleFinalDataOfCreatePost.php

```diff
@@ -159,7 +159,26 @@
 		echo "<br><br><h3 style = 'color:white ;'> $post Post  already exists .</h3>";
 		die ();
 	}
-
+	
+	// create table for asigning posts
+	$stmt = $sqlConn->prepare("CREATE TABLE IF NOT EXISTS $assignPostTable ($postTitle VARCHAR(100) NOT NULL, $assignedUser VARCHAR(100), INDEX idx USING BTREE ($postTitle))");
+	if ( ! $stmt->execute() )
+	{
+		echo"there is a problem with database<br>";
+		die ();
+	}
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->